src/main/java/step3/InputView.java

```diff
@@ -0,0 +1,48 @@
+package step3;
+
+import static step3.ResultView.printCarCount;
+import static step3.ResultView.printTryCount;
+
+import java.util.InputMismatchException;
+import java.util.Scanner;
+
+public class InputView {
+    public static int inputCountCar() {
+        Scanner scanner = new Scanner(System.in);
+        int carCount = 0;
+        boolean isValid = false;
+        while (!isValid) {
+            printCarCount();
+            try {
+                carCount = scanner.nextInt();
+                isValid = isPositiveNum(carCount);
+            } catch (InputMismatchException ex) {
+                scanner.next();
+            }
+        }
+        return carCount;
+    }
+
+    public static boolean isPositiveNum(int num) {
+        if (num <= 0) {
+            return false;
+        }
+        return true;
```

Comment:
```suggestion
        return num > 0;
```

이렇게 바꾸는것도 방법이 될것 같아요!

Reply:
더 간단한 방법이 있었네요!